Libraries/Utilities/BackHandler.ios.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * On Apple TV, this implements back navigation using the TV remote's menu button.
+ * On iOS, this just implements a stub.
+ *
+ * @providesModule BackHandler
+ */
+
+'use strict';
+
+var Platform = require('Platform');
+var TVEventHandler = require('TVEventHandler');
+
+type BackPressEventName = $Enum<{
```

Comment:
Wondering why it's not `type BackPressEventName = 'backPress'` instead

Also seems all examples say that the event name is `hardwareBackPress`, while the type says `backPress`